tests/functional/z_cluster/test_five_mon.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.pytest_customization.marks import (
+    brown_squad,
+    ignore_leftovers,
+    skipif_less_than_five_workers,
+)
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_ocs_version,
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility import prometheus
+from ocs_ci.ocs.resources.pod import verify_mon_pod_running
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@ignore_leftovers
```

Comment:
any specific reason for leftover marker?

Reply:
As the test updates mon count to five and we do not have option to scale down for now, added ignore leftover label to address teardown failures